src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -178,10 +180,16 @@ def volume_info(self, **kwargs):
                         pool_type = "metadata"
                     else:
                         pool_type = "data"
-                    vol_info_dict["pools"][pool_type].append({
-                                    'name': pools[pool_id]['pool_name'],
-                                    'used': pool_stats[pool_id]['bytes_used'],
-                                    'avail': pool_stats[pool_id]['max_avail']})
+                    if not flag:
```

Comment:
s/flag/human_readable ?  This makes code reading easier.

Reply:
I feel `if human_readable` (`if flag` for now) would be better than using `not`. Because the intention here is to call that block of code when we pass `--human_readable`. `if not flag` looks a bit odd here. Just a personal opinion.

Reply:
Done!!